README.md

```diff
@@ -98,6 +98,8 @@ A number of improvements were made to the pipeline recently, mainly with regard
 
 4. Start running your own analysis!
 
+   > - Please provide pipeline parameters via the CLI or Nextflow `-params-file` option. Custom config files including those provided by the `-c` Nextflow option can be used to provide any configuration except for parameters; see [docs](https://nf-co.re/usage/configuration#custom-configuration-files).
```

Comment:
Whatever is written in the pipeline README is rendered on the website?

I agree with trimming down the boilerplate but this is quite important. A number of users have been stung by it and complained that there is no clear documentation for this in the pipeline. Think we should have it on the website in more detail as well as the pipeline in as many prominent locations as possible. Not sure we should compromise on this if it saves a user countless hours figuring out why their parameters are not being passed to the pipeline.

Reply:
See https://github.com/nf-core/rnaseq/issues/894 for context

Another user spoke to me about this at the Festival of Genomics.

Reply:
I'm scrambling to get 2 pipelines out this week and have a bunch of other stuff I need to do before the Hackathon. At the very least I will create an issue pointing to the commits so it can be easily added during the Hackathon.

Once we have revised the website docs we can always trim down the text but let's keep it as it is for now.

Reply:
I think it would also be possible to add a check for this, see 
https://github.com/nf-core/tools/issues/2184